tests/kafkatest/services/simple_consumer_shell.py

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.services.background_thread import BackgroundThreadService
+
+from kafkatest.services.kafka.directory import kafka_dir
+from kafkatest.services.security.security_config import SecurityConfig
+
+
+class SimpleConsumerShell(BackgroundThreadService):
+
+    logs = {
+        "simple_consumer_shell_log": {
+            "path": "/mnt/simple_consumer_shell.log",
+            "collect_default": False}
+    }
+
+    def __init__(self, context, num_nodes, kafka, topic, partition=0, security_protocol="PLAINTEXT"):
+        super(SimpleConsumerShell, self).__init__(context, num_nodes)
+
+        self.kafka = kafka
+        self.topic = topic
+        self.partition = partition
+        self.security_protocol = security_protocol
```

Comment:
Since this is simple consumer, I think security is not applicable here? We can probably remove security-related fields here.